web/wp-content/themes/dctx/index.php

```diff
@@ -1,21 +1,25 @@
 <?php get_header(); ?>
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-ntyjwn.php, but found phpcs-scan-NTYjwn.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).